lib/init.ts

```diff
@@ -61,7 +62,7 @@ export function init(options: InitParams) {
   this._endpointOrigin = options.region
     ? `https://insights.${options.region}.algolia.io`
     : "https://insights.algolia.io";
-
+  this._useCookie = options.useCookie ?? true;
```

Comment:
What's the difference between `options.useCookie` vs `options.userHasOptedOut` ?

Reply:
If user denies to be tracked, for example, by clicking "deny" button on a cookie popup, then `userHasOptedOut` should be `true`, and `search-insights` won't send any event from the user. So this is per-user setting.

On the other hand, if a website decides not to use anonymous user token in cookie at all, then they can set `useCookie: false`. It means `search-insights` will not generate/store any anonymous user token in the cookie. So `aa` will not have any user token initially. `setUserToken` must be called explicitly. Once a userToken is set, it will send events.

Does it make sense to you? Let me know if anything is unclear.

Reply:
Yep that make sense!